Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Kimai - Time tracking/Main application](https://hosted.weblate.org/projects/kimai/messages/).


It also includes following components:

* [Kimai - Time tracking/Emails](https://hosted.weblate.org/projects/kimai/emails/)

* [Kimai - Time tracking/Export](https://hosted.weblate.org/projects/kimai/export/)

* [Kimai - Time tracking/User feedback (Flash messages and Alerts)](https://hosted.weblate.org/projects/kimai/flashmessages/)

* [Kimai - Time tracking/Invoice-Numbers](https://hosted.weblate.org/projects/kimai/invoice-numbers/)

* [Kimai - Time tracking/Invoice-Templates](https://hosted.weblate.org/projects/kimai/invoice-templates/)

* [Kimai - Time tracking/Reporting](https://hosted.weblate.org/projects/kimai/reporting/)

* [Kimai - Time tracking/System configurations](https://hosted.weblate.org/projects/kimai/system-configuration/)

* [Kimai - Time tracking/Invoice-Calculator](https://hosted.weblate.org/projects/kimai/invoice-calculator/)

* [Kimai - Time tracking/Wizard](https://hosted.weblate.org/projects/kimai/wizard/)

* [Kimai - Time tracking/Validations](https://hosted.weblate.org/projects/kimai/validations/)

* [Kimai - Time tracking/Teams](https://hosted.weblate.org/projects/kimai/teams/)

* [Kimai - Time tracking/Actions (Dropdowns and Toolbars)](https://hosted.weblate.org/projects/kimai/actions/)

* [Kimai - Time tracking/About](https://hosted.weblate.org/projects/kimai/about/)

* [Kimai - Time tracking/Date-Picker](https://hosted.weblate.org/projects/kimai/daterangepicker/)

* [Kimai - Time tracking/Plugins](https://hosted.weblate.org/projects/kimai/plugins/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/kimai/-/messages/horizontal-auto.svg)

kevinpapst reviewed (changes requested):
filenames for locales with region are wring

weblate committed:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (345 of 345 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 51.3% (177 of 345 strings)

Translated using Weblate (Greek)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Faroese)

Currently translated at 71.3% (246 of 345 strings)

Translated using Weblate (Finnish)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Portuguese)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Turkish)

Currently translated at 99.7% (344 of 345 strings)

Translated using Weblate (Swedish)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Slovak)

Currently translated at 74.4% (257 of 345 strings)

Translated using Weblate (Russian)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Romanian)

Currently translated at 82.0% (283 of 345 strings)

Translated using Weblate (Polish)

Currently translated at 83.7% (289 of 345 strings)

Translated using Weblate (Dutch)

Currently translated at 77.3% (267 of 345 strings)

Translated using Weblate (Japanese)

Currently translated at 55.3% (191 of 345 strings)

Translated using Weblate (Hungarian)

Currently translated at 83.7% (289 of 345 strings)

Translated using Weblate (Basque)

Currently translated at 68.9% (238 of 345 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (345 of 345 strings)

Translated using Weblate (Esperanto)

Currently translated at 81.7% (282 of 345 strings)

Translated using Weblate (German (Switzerland))

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Danish)

Currently translated at 64.0% (221 of 345 strings)

Translated using Weblate (Czech)

Currently translated at 82.0% (283 of 345 strings)

Translated using Weblate (English)

Currently translated at 100.0% (345 of 345 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (345 of 345 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 51.0% (176 of 345 strings)

Translated using Weblate (Greek)

Currently translated at 81.7% (282 of 345 strings)

Translated using Weblate (Faroese)

Currently translated at 70.7% (244 of 345 strings)

Translated using Weblate (Finnish)

Currently translated at 81.7% (282 of 345 strings)

Translated using Weblate (Portuguese)

Currently translated at 82.0% (283 of 345 strings)

Translated using Weblate (Turkish)

Currently translated at 99.4% (343 of 345 strings)

Translated using Weblate (Swedish)

Currently translated at 82.0% (283 of 345 strings)

Translated using Weblate (Slovak)

Currently translated at 74.2% (256 of 345 strings)

Translated using Weblate (Russian)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Polish)

Currently translated at 83.1% (287 of 345 strings)

Translated using Weblate (Japanese)

Currently translated at 54.7% (189 of 345 strings)

Translated using Weblate (Italian)

Currently translated at 81.7% (282 of 345 strings)

Translated using Weblate (Hungarian)

Currently translated at 83.7% (289 of 345 strings)

Translated using Weblate (Basque)

Currently translated at 68.6% (237 of 345 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (345 of 345 strings)

Translated using Weblate (Esperanto)

Currently translated at 81.1% (280 of 345 strings)

Translated using Weblate (German (Switzerland))

Currently translated at 81.7% (282 of 345 strings)

Translated using Weblate (Danish)

Currently translated at 63.7% (220 of 345 strings)

Translated using Weblate (Czech)

Currently translated at 81.7% (282 of 345 strings)

Translated using Weblate (English)

Currently translated at 100.0% (345 of 345 strings)

Translated using Weblate (German (Switzerland))

Currently translated at 3.5% (1 of 28 strings)

Translated using Weblate (Faroese)

Currently translated at 46.4% (13 of 28 strings)

Translated using Weblate (Portuguese)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Greek)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Turkish)

Currently translated at 100.0% (28 of 28 strings)

Translated using Weblate (Swedish)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Slovak)

Currently translated at 82.1% (23 of 28 strings)

Translated using Weblate (Romanian)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Polish)

Currently translated at 100.0% (28 of 28 strings)

Translated using Weblate (Italian)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Hungarian)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Basque)

Currently translated at 82.1% (23 of 28 strings)

Translated using Weblate (Esperanto)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Czech)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Finnish)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (German (Switzerland))

Currently translated at 0.0% (0 of 28 strings)

Translated using Weblate (Faroese)

Currently translated at 42.8% (12 of 28 strings)

Translated using Weblate (Portuguese)

Currently translated at 85.7% (24 of 28 strings)

Translated using Weblate (Greek)

Currently translated at 85.7% (24 of 28 strings)

Translated using Weblate (Finnish)

Currently translated at 85.7% (24 of 28 strings)

Translated using Weblate (Turkish)

Currently translated at 96.4% (27 of 28 strings)

Translated using Weblate (Swedish)

Currently translated at 85.7% (24 of 28 strings)

Translated using Weblate (Slovak)

Currently translated at 78.5% (22 of 28 strings)

Translated using Weblate (Romanian)

Currently translated at 85.7% (24 of 28 strings)

Translated using Weblate (Polish)

Currently translated at 96.4% (27 of 28 strings)

Translated using Weblate (Hungarian)

Currently translated at 85.7% (24 of 28 strings)

Translated using Weblate (Basque)

Currently translated at 78.5% (22 of 28 strings)

Translated using Weblate (Esperanto)

Currently translated at 85.7% (24 of 28 strings)

Translated using Weblate (Czech)

Currently translated at 85.7% (24 of 28 strings)

Translated using Weblate (Hungarian)

Currently translated at 83.7% (289 of 345 strings)

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (German (Switzerland))

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (Slovak)

Added translation using Weblate (Korean)

Added translation using Weblate (Faroese)

Added translation using Weblate (Finnish)

Added translation using Weblate (Swedish)

Added translation using Weblate (Dutch)

Added translation using Weblate (Arabic)

Added translation using Weblate (Hungarian)

Added translation using Weblate (Vietnamese)

Added translation using Weblate (Portuguese)

Added translation using Weblate (Czech)

Added translation using Weblate (Japanese)

Added translation using Weblate (Romanian)

Added translation using Weblate (Russian)

Added translation using Weblate (Basque)

Added translation using Weblate (Esperanto)

Added translation using Weblate (Greek)

Added translation using Weblate (Danish)

Added translation using Weblate (Chinese (Traditional))

Translated using Weblate (Chinese (Traditional))

Currently translated at 51.3% (177 of 345 strings)

Translated using Weblate (German (Switzerland))

Currently translated at 0.0% (0 of 28 strings)

Translated using Weblate (Faroese)

Currently translated at 42.8% (12 of 28 strings)

Translated using Weblate (Portuguese)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Greek)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Greek)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Faroese)

Currently translated at 71.3% (246 of 345 strings)

Translated using Weblate (Finnish)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Finnish)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Portuguese)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Turkish)

Currently translated at 100.0% (28 of 28 strings)

Translated using Weblate (Swedish)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Slovak)

Currently translated at 82.1% (23 of 28 strings)

Translated using Weblate (Romanian)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Polish)

Currently translated at 100.0% (28 of 28 strings)

Translated using Weblate (Dutch)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Hungarian)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Basque)

Currently translated at 82.1% (23 of 28 strings)

Translated using Weblate (Esperanto)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Danish)

Currently translated at 67.8% (19 of 28 strings)

Translated using Weblate (Czech)

Currently translated at 89.2% (25 of 28 strings)

Translated using Weblate (Turkish)

Currently translated at 100.0% (345 of 345 strings)

Translated using Weblate (Swedish)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Slovak)

Currently translated at 74.7% (258 of 345 strings)

Translated using Weblate (Romanian)

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Polish)

Currently translated at 83.7% (289 of 345 strings)

Translated using Weblate (Dutch)

Currently translated at 77.6% (268 of 345 strings)

Translated using Weblate (Japanese)

Currently translated at 55.3% (191 of 345 strings)

Translated using Weblate (Hungarian)

Currently translated at 79.7% (275 of 345 strings)

Translated using Weblate (Basque)

Currently translated at 68.9% (238 of 345 strings)

Translated using Weblate (Esperanto)

Currently translated at 81.7% (282 of 345 strings)

Translated using Weblate (German (Switzerland))

Currently translated at 82.3% (284 of 345 strings)

Translated using Weblate (Danish)

Currently translated at 64.0% (221 of 345 strings)

Translated using Weblate (Czech)

Currently translated at 82.3% (284 of 345 strings)

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (Chinese (Traditional))

Added translation using Weblate (Chinese (Traditional))

Translated using Weblate (Arabic)

Currently translated at 100.0% (8 of 8 strings)

Translated using Weblate (Arabic)

Currently translated at 88.4% (23 of 26 strings)

Translated using Weblate (Arabic)

Currently translated at 94.2% (325 of 345 strings)

Co-authored-by: Anonymous <noreply@weblate.org>
Co-authored-by: ButterflyOfFire <ButterflyOfFire@protonmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kevin Papst <kevin@kevinpapst.de>
Co-authored-by: Kis Momesz <kismomesz@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/cs/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/de_CH/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/el/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/eo/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/eu/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/fi/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/fo/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/hu/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/it/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/nl/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/pl/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/pt/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/ro/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/sk/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/sv/
Translate-URL: https://hosted.weblate.org/projects/kimai/actions/tr/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/cs/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/da/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/de_CH/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/el/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/en/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/eo/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/es/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/eu/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/fi/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/fo/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/hu/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/it/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ja/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/nl/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/pl/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/pt/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ro/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ru/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/sk/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/sv/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/tr/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/uk/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/kimai/teams/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/ar/
Translation: Kimai - Time tracking/Actions (Dropdowns and Toolbars)
Translation: Kimai - Time tracking/Main application
Translation: Kimai - Time tracking/Teams
Translation: Kimai - Time tracking/Validations
